### PR TITLE
log SSM command ID in e2e test run outcome

### PR DESCRIPTION
**Pull request by danbudris**
*Description of changes:*
bubble up the SSM command id from the execution of each test run, and include it in the final log message for completed tests. This is useful for indexing various things include the CW log streams generated by each command

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/562#" title="Author self-approved">danbudris</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [danbudris]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: internal/test/e2e/run.go
Comment (g-gaston) [inline internal/test/e2e/run.go:86]: I'm not sure `commandId` makes a lot of sense in this struct
This is supposed to be the input for an instance run. The command id seems more like the output

I think it would make more sense in `instanceTestsResults`

File: internal/test/e2e/setup.go
Comment (g-gaston) [inline internal/test/e2e/setup.go:37]: This is a bit confusing
Each session could potentially run multiple commands
I think this `commandId` is more like part of the output of running a test (or multiple tests), I don't think it belongs in this struct

File: internal/test/e2e/run.go
Comment (g-gaston) [inline internal/test/e2e/run.go:131]: Instead of storing it in the session I think the `commandID` can just be another return value from this method

Comment (danbudris) [reply internal/test/e2e/run.go:131]: yeah that's fair

**Review by g-gaston — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/562#" title="Author self-approved">danbudris</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/562#pullrequestreview-796689573" title="LGTM">g-gaston</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [danbudris,g-gaston]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
